src/index.js

```diff
@@ -0,0 +1,74 @@
+import './style/main.css';
+import deleteImage from './img/delete.svg';
+
+import {
+  addTodo, deleteTodo, clearAllComplele, checkTodo, updateDescription, refreshPage,
+} from './utils.js';
+
+const myWrapper = document.getElementById('list-wrapper');
+const todoInput = document.getElementById('todo-input');
+
+document.getElementById('refresh').addEventListener('click', () => refreshPage());
+
+todoInput.addEventListener('keyup', (e) => {
+  if (e.key === 'Enter') {
+    addTodo(e.target.value);
+  }
+});
+
+const todos = JSON.parse(localStorage.getItem('todos'))
+  ? JSON.parse(localStorage.getItem('todos'))
+  : [];
+todos.forEach((todo) => {
+  const listItem = document.createElement('li');
+  listItem.classList.add('list-item');
+  const itemContent = document.createElement('input');
+  const checkBox = document.createElement('input');
+  const moreIcon = document.createElement('img');
+  const myContentWrapper = document.createElement('div');
+  myContentWrapper.classList.add('ct-wrapper');
+  moreIcon.classList.add('icon-cust');
+  moreIcon.setAttribute('src', `${deleteImage}`);
+  moreIcon.setAttribute('width', '20');
+  moreIcon.setAttribute('heigt', '20');
```

Comment:
**DRY code**
Good work on breaking up your code into as many chunks of reusable code as possible so that your code is DRY. However don't you think it would be a good idea to place this functionality inside a function to further make your code DRY?